packages/page-bounties/src/VotingDescription.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright 2017-2021 @polkadot/app-bounties authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { DeriveCollectiveProposal } from '@polkadot/api-derive/types';
+import type { BountyStatus } from '@polkadot/types/interfaces';
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { getVotingDescription } from './helpers/extendedStatuses';
+import { useTranslation } from './translate';
+
+interface Props {
+  className?: string;
+  proposals: DeriveCollectiveProposal[];
+  status: BountyStatus;
+}
+
+function VotingDescription ({ className = '', proposals, status }: Props): JSX.Element {
```

Comment:
```suggestion
function VotingDescription ({ className = '', proposals, status }: Props): React.ReactElement<Props> {
```

Just aligns with what we do everywhere